SU-HSDO/suhumsci
# STN-321 Sand Color Pairing Update

# READY FOR REVIEW

## Summary
Updated the Sand color pairing to have the appropriate Stanford branding color palette. Colors have been approved by a designer and H&S. Link to Jira card can be found here: https://sparkbox.atlassian.net/browse/STN-321.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. In the CLI, run npm run test. Verify all tests pass
2. To test the Sand color pairing in Drupal, go to Appearance > Settings > HumSci Colorful, scroll down to Color Pairing, choose Sand pairing and then Save Configuration.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
